docs/pages/includes/tctl.mdx

```diff
@@ -1,44 +1,31 @@
-<Details 
-title="Make sure you can connect to Teleport" 
-scope={["oss", "enterprise"]} 
-scopeOnly={true}
-opened={true}
->
-
-To connect to Teleport, log in to your cluster using `tsh`, then use `tctl`
+- Make sure you can connect to Teleport. Log in to your cluster using `tsh`, then use `tctl`
```

Comment:
What would you think of adding the bullet where we include `tctl.mdx`, rather than inside the partial? That way, we can also include the partial outside of bulleted lists if we need to.

Reply:
I'm a big fan of this idea, it's what I was trying to discuss in #20308. It's outside the scope of this PR since we'll need to review every page the partial is used in, but I think we should make a separate issue for this work.

Reply:
Unresolving because I've come around to the idea of just doing it here and now.

Reply:
Done.